project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ object Version {
   val akka       = "2.3.9"
   val akkaHttp   = "1.0-M4"
   val junit      = "4.12"
-  val scala      = "2.11.6"
+  val scala      = "2.11.5"
```

Comment:
I think we should add the resolver here then right? Otherwise people are going to find it hard to contribute easily. :-)

...and then there's CI...